.github/policies/resourceManagement.yml

```diff
@@ -0,0 +1,115 @@
+id: 
+name: GitOps.PullRequestIssueManagement
+description: GitOps.PullRequestIssueManagement primitive
+owner: 
+resource: repository
+disabled: false
+where: 
+configuration:
+  resourceManagementConfiguration:
+    scheduledSearches:
+    - description: Auto close "Area-NuGet" issues and add a comment pointing to NuGet repo
+      frequencies:
+      - daily:
+          time: 16:0
+      filters:
+      - isIssue
+      - isOpen
+      - hasLabel:
+          label: Area-NuGet
+      actions:
+      - addReply:
+          reply: >-
+            Thanks for creating this issue! We believe this issue is related to NuGet tooling, which is maintained by the NuGet team. Thus, we closed this one and encourage you to raise this issue in the [NuGet repository](https://github.com/NuGet/Home) instead. Don’t forget to check out [NuGet’s contributing guide](https://github.com/NuGet/Home/blob/dev/CONTRIBUTING.md#before-submitting-an-issue) before submitting an issue!
+
+
+            If you believe this issue was closed out of error, please comment to let us know.
+
+
+            Happy Coding!
+      - closeIssue
+    eventResponderTasks:
+    - if:
+      - payloadType: Pull_Request
+      - hasLabel:
+          label: Auto-Merge If Tests Pass
+      then:
+      - enableAutoMerge:

```

Comment:
by default does this ensure all checks are passing?

Reply:
That's right. This will enable GitHub's built-in auto-merge behavior, where it will auto-merge if all requirements are met the same as would happen if you click the 'Enable auto-merge (squash)' button on the PR. By applying the https://github.com/dotnet/sdk/labels/Auto-Merge%20If%20Tests%20Pass label, auto-merge will be enabled; removing the label will disable auto-merge of the PR.

This rule was carried over from the previous configuration from FabricBot, but it doesn't look like it's been used the last couple of years.